framework/CodeInterfaces/PHISICS/combine.py

```diff
@@ -0,0 +1,173 @@
+          raise ValueError("\n the keyword time is not listed in the RELAP csv output \n")
+    
+  def selectPhisicsTime(self,depTimeDict):
+    """
+      Selects the time (<=> line) that will be printed in the final RAVEN output. Those times match with the RELAP one.
+      @ In, depTimeDict, dictionary, information from the xml depletion file
+      @ Out, None
+    """
+    timeList = []
+    self.timeStepSelected = []
+    lineSelected = 0
+    for i in depTimeDict['timeSteps'].split(' '):
+      lineSelected = lineSelected + int(i)
+      self.timeStepSelected.append(lineSelected)
+      lineNumber = 0
+      with open(self.phisicsCSV, 'r') as phifile:
+        for line in phifile:
+          if lineNumber == lineSelected:
+            timeList.append(line.split(',')[0])
+          lineNumber = lineNumber + 1
+    return timeList
+  
+  def putCSVinDict(self,csvFile):
+    """
+      Places each line of the csv into a dictionary. key: line number (integer), value: line (list).
+      @ In, csvFile, string, file name of the csv file parsed
+      @ Out, csvDict, dictionary
+    """
+    countLine = 0
+    csvDict = {}
+    with open(csvFile, 'r') as file:
+      for line in file:
+        if countLine == 0 and csvFile == self.phisicsCSV:
+          self.numOfParameters = self.getNumOfParameters(line)
+        countLine = countLine + 1
+        csvDict[countLine] = line
+    return csvDict
+  
+  def getNumOfParameters(self,line):
+    """
+      Returns the number of parameters that are on one line of the csv file.
+      @ In, line, string, csv file first line
+      @ Out, getNumOfParameters, integer
+    """
+    return len(line.split(','))
+
+  def getNumOfLines(self,csvFile):
+    """
+      Counts the number of lines in the PHISICS or RELAP csv.
+      @ In, csvFile, string, csv file name
+      @ Out, count, integer
+    """
+    count = 0
+    with open(csvFile, 'r') as file:
+      for line in file:
+        count = count + 1
+    return count
+  
+  def joinLine(self,phisicsList,relapList):
+    """
+      Joins the PHISICS and RELAP line into one list.
+      @ In, phisicsList, list, list contianing a csv PHISICS line
+      @ In, relapList, list, list containing a csv RELAP line
+      @ Out, joinedList, list, joined list of parameters
+    """
+    joinedList = []
+    joinedList.append(phisicsList.strip('\n').strip('\r'))
+    joinedList.append(relapList.strip('\n').strip('\r'))
+    return joinedList
+  
+  def joinCSV(self,paramDict):
+    """
+      Joins the RELAP csv and PHISICS csv based on the time lines selected from PHISICS.
+      @ In, paramDict, dictionary, dictionary of parameters
+      @ Out, None
+    """
+    with open(os.path.join(self.workingDir,'dummy.csv'), 'wb') as f:
+      instantWriter = csv.writer(f, delimiter=str(u',').encode('utf-8'),quotechar=str(u' ').encode('utf-8'), quoting=csv.QUOTE_MINIMAL)
+      instantWriter.writerow(self.joinLine(paramDict['phiDict'][1],paramDict['relDict'][1]))
```

Comment:
joinLine is useless

Reply:
No why do you say that?
I tried the joinLine without `strip('\n').strip('\r')` the phisicsList and relapLine end up not being printed on the same line. 
nothing is printed if I simply do 
instantWriter.writerow([paramDict['phiDict'][1]] + [paramDict['relDict'][1]])

Reply:
the method ```joinLine``` is not useful 

```rstrip()``` does what you want.

Reply:
i.e.

```
instantWriter.writerow(self.joinLine(paramDict['phiDict'][1],paramDict['relDict'][1]))
```
to
```
instantWriter.writerow( [paramDict['phiDict'][1].rstrip(),paramDict['relDict'][1].rstrip() ])
```